shell/platform/darwin/macos/framework/Source/KeyCodeMap.mm

```diff
@@ -34,14 +34,6 @@
  */
 const uint64_t kUnicodePlane = 0x00000000000;
 
-/**
- * Mask for the auto-generated bit portion of the key code.
- *
- * This is used by platform-specific code to generate new Flutter key codes for
- * keys which are not recognized.
- */
-const uint64_t kAutogeneratedMask = 0x10000000000;
```

Comment:
(I'm aware that I should have used a design doc to discuss it first. My apology. If deemed necessary I can create one now, including some other changes that has or will come.)

Reply:
It should only be auto-generated if we don't have it in our database, because
once it's in the database, we don't change the ID. The autogenerated bit is
there to indicate that the value needs to be compared in a platform-specific
way if you want to look for it, because we haven't set that value in stone yet
by including it in our database.

Here's the definition of the two planes from the design with more description of why:

Platform planes:
- **0x02 0000 0000 - 0xFF FFFF FFFF**: For keys that aren't defined in USB at the
  time of implementation, but that we need to support. For example, if Flutter
  were ever ported to the Symbolics LM-2, the "thumb up" key might be given
  the code 0x14 0000 0001, where 0x14 is defined as the “Symbolics” platform
  range. Where possible, we will use specific subranges of this space to reuse
  keys from other platforms. When this is not possible, the prefix 0xFF is
  reserved for “Custom” codes. Each platform from which we take codes will get
  a unique prefix in the range 0x2-0xFE. If multiple systems define keys with
  the same usage (not the same number), then the value with the lowest prefix
  is used as the defining code.

  Prefixes will be:

  | Code | Platform |
  | ---- | -------- |
  | 0x02 | Android  |
  | 0x03 | Fuchsia  |
  | 0x04 | iOS      |
  | 0x05 | macOS    |
  | 0x06 | Linux    |
  | 0x07 | Windows  |
  | 0x08 | Web      |
  | 0xFF | Custom   |

  Further ranges will be added as platforms are added. The platform prefix
  does not define the platform it is used on, it is just the platform that
  decides what the value is: the codes are mapped to the same value on all
  platforms.

Auto-generated plane:
- **0x100 0000 0000 - 0x1FF FFFF FFFF**: For keys that have no definition yet in
  Flutter, but that are encountered in the field, this range is used to embed
  the platform-specific keycode in an ID that must be tested for in a
  platform-specific way. For instance, if a platform generates a new USB
  HID code 0x07 00E8 that a Flutter app wasn’t compiled with, then it would
  appear in the app as 0x100 0007 00E8, and the app could test against that
  code. Yes, this also means that once they recompile with a version of
  Flutter that supports this new HID code, apps looking for this code will
  break. This situation is only meant to provide a fallback ability for apps
  to handle esoteric codes that their version of Flutter doesn’t support yet.
  The prefix for this code is the platform prefix from the previous sections,
  plus 0x100.

I don't think you need a whole new design doc if it's just this change, but it might be a good idea to add this if you have a few changes to discuss.